src/stackdriver_agent.c

```diff
@@ -119,10 +121,17 @@ static int sagt_read(user_data_t *user_data)
   value_list_t vl = {};  // zero-init
   sstrncpy(vl.plugin, this_plugin_name, sizeof(vl.plugin));
 
+  // This functions passes metric information using two mechanisms. The standard
```

Comment:
Typo: functions.

Reply:
Thanks. Next time feel free to edit the code directly and just LGTM the change.